canvas_modules/common-canvas/src/common-canvas/svg-canvas-renderer.js

```diff
@@ -919,7 +919,7 @@ export default class SVGCanvasRenderer {
 	// Highlights any data link, that an 'insertable' nodeTemplate from the
 	// palette, is dragged over. The x and y passed in are in page coordinates
 	// based on the top left corner of the page.
-	paletteNodeDraggedOver(nodeTemplate, x, y) {
+	nodeTemplateDraggedOver(nodeTemplate, x, y) {
```

Comment:
I found references of `paletteNodeDraggedOver` in `svg-canvas-d3.js`, those should be renamed as well

Reply:
Thanks!